src/const.ts

```diff
@@ -224,10 +224,11 @@ export const DISABLED_TOKEN_MAPS = Object.keys(disabledTokens).reduce<DisabledTo
     [Network.MAINNET]: {},
     [Network.RINKEBY]: {},
     [Network.GNOSIS_CHAIN]: {},
+    [Network.GOERLI]: {},
   },
 )
 
-export const COW_SDK = [Network.MAINNET, Network.RINKEBY, Network.GNOSIS_CHAIN].reduce<
+export const COW_SDK = [Network.MAINNET, Network.RINKEBY, Network.GNOSIS_CHAIN, Network.GOERLI].reduce<
```

Comment:
with the latest version we can omit this and have just 1 instance instead of 4.

Then we can pass the chainId as an `options` object to each method of the cowApi class